src/qt/configurenamedialog.h

```diff
@@ -0,0 +1,45 @@
+#ifndef CONFIGURENAMEDIALOG_H
+#define CONFIGURENAMEDIALOG_H
+
+#include "platformstyle.h"
+
+#include <QDialog>
+
+namespace Ui {
+    class ConfigureNameDialog;
+}
+
+class WalletModel;
+
+/** Dialog for editing an address and associated information.
+ */
+class ConfigureNameDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+
+    explicit ConfigureNameDialog(const PlatformStyle *platformStyle,
+                                 const QString &_name, const QString &data,
+                                 bool _firstUpdate, QWidget *parent = 0);
+    ~ConfigureNameDialog();
+
+    void setModel(WalletModel *walletModel);
+    const QString &getReturnData() const { return returnData; }
+
+public Q_SLOTS:
+    void accept();
+    void reject();
+    void on_addressBookButton_clicked();
+    void on_pasteButton_clicked();
+
+private:
+    Ui::ConfigureNameDialog *ui;
```

Comment:
Could you make this a non-pointer member instead?  I'm not very experienced in Qt and do not know if this is a common pattern, but a very brief look at the code suggests that you could do that.  This is in general preferrable to pointers and lets you get rid of the destructor.  The same may (or may not) be true for the other members as well.